<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: bold;</string>
   </property>
   <widget class="QLineEdit" name="input_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
font: 22px bold;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_del">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="numbers" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(251, 206, 255);
}
QPushButton:hover {
    background-color: rgb(135, 97, 255);
}
QPushButton:pressed {
    background-color: rgb(255, 22, 228);
}
</string>
    </property>
    <widget class="QPushButton" name="button_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_dot">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_0">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 1px solid gray;
	background-color: #FBCEFF;
}
QPushButton:hover {
    background-color: #8761FF;
}
QPushButton:pressed {
    background-color: #FF16E4;
}
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_unary">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_atan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ln">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_x">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_acos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mod">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sqrt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close_br">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_power">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>xʸ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open_br">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QWidget" name="operations" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>41</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(4, 51, 255);
font: 26px;</string>
    </property>
    <widget class="QPushButton" name="button_mul">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string extracomment="*">×</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_equal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_minus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_plus">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_div">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>÷</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_ac">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: normal;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="x_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>X=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_min">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_max">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_min">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Xmin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_max">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Xmax</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_min">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_max">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_min">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ymin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_max">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ymax</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plot">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_plot">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreditButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DepositButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
